docs/roadmap.md

```diff
@@ -3,14 +3,18 @@
 - [Roadmap](#roadmap)
   - [v2.3](#v23)
     - [Merge Argo CD Notifications into Argo CD](#merge-argo-cd-notifications-into-argo-cd)
-    - [Input Forms UI Refresh](#input-forms-ui-refresh)
+    - [Merge ApplicationSet controller into Argo CD](#merge-applicationset-controller-into-argo-cd)
     - [Compact resources tree](#compact-resources-tree)
     - [Maintain difference in cluster and git values for specific fields](#maintain-difference-in-cluster-and-git-values-for-specific-fields)
-    - [Web Shell](#web-shell)
-    - [Helm values from external repo](#helm-values-from-external-repo)
+    - [ARM images and CLI binary](#arm-images-and-cli-binary)
+    - [Server side apply](#server-side-apply)
```

Comment:
Are we sure we get that into 2.3? From my PoV after looking at the PRs, there might be necessity for some more design work around that.